hivemind/p2p/p2p_daemon.py

```diff
@@ -28,7 +30,6 @@ class P2PContext(object):
     handle_name: str
     local_id: PeerID
     remote_id: PeerID = None
-    remote_maddr: Multiaddr = None
```

Comment:
To future reviewers: I and @deniskamazur have agreed to remove this because

1) Handlers are supposed to work with peer IDs only (not with multiaddrs, which is a lower-level concept)
2) Absence of `remote_maddr` simplifies the unary handler implementation